Godeps/Godeps.json

```diff
@@ -1964,6 +1964,10 @@
 			"Rev": "5cbbc6bafb45bd7ef10486b673deb3b81bb3b787"
 		},
 		{
+			"ImportPath": "github.com/mtrmac/gpgme",
```

Comment:
This is not an OpenShift-specific fork; `containers/image` and `projectatomic/skopeo` use the same one.  See https://github.com/proglottis/gpgme/issues/6 for the full history behind creating it. Hopefully it will eventually go away.

If OpenShift _wants_ to take over that fork and commit to maintaining it, sure, why not. I don’t see how OpenShift benefits (especially because OpenShift opts out of actually using anything in this package, using the `containers_image_openpgp` build tag), but I’d be only too happy to hand over this responsibility :)